onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -42,6 +53,33 @@ class OpenDataViewSet(ETagsMixin, CacheControlMixin,
     MAX_INSTANCES_PER_REQUEST = 1000
     pagination_class = StandardPageNumberPagination
 
+    def clean_data_fields(self, data, xform):  # pylint disable=no-self-use
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5447927)